packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -594,7 +594,7 @@ class ScrollableBase extends Component {
 						this.checkAndApplyOverscrollEffectByDirection(direction);
 					}
 				}
-			} else if (getDirection(keyCode)) {
+			} else if (!Spotlight.getPointerMode() && getDirection(keyCode)) {
```

Comment:
I'm confusing that it is possible to catch 5-way key input in pointer mode here. Is `onKeyDown` handler called before the mode is changed to 5-way mode?

Reply:
Let's investigate this one more and apply it again if needed.